examples/js/loaders/3MFLoader.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @author technohippy / https://github.com/technohippy
+ */
+
 THREE.ThreeMFLoader = function ( manager ) {
```

Comment:
OK, that makes sense. But it's still not very intuitive - I had gone over the loaders quite a few times before I realised there was a 3ds loader. I just assumed that tds was some obscure format I had never heard of.